packages/babel-preset-react-app/index.js

```diff
@@ -84,10 +84,13 @@ if (env === 'test') {
   module.exports = {
     presets: [
       // Latest stable ECMAScript features
-      [require.resolve('babel-preset-latest'), {
-        'es2015': {
-          modules: false
-        }
+      [require.resolve('babel-preset-env'), {
+        targets: {
+          // We support everything React supports
+          'ie': 9
```

Comment:
I understand but it's good to leave on for people who get excited and immediately override this after ejecting. 😉